.github/workflows/ci.yml

```diff
@@ -12,98 +12,144 @@ concurrency:
 # ${{ vars.CI_UNIFIED_IMAGE }} is defined in the repository variables
 
 jobs:
-  check:
-    name: "check"
+  fmt:
+    name: Cargo fmt
     runs-on: ubuntu-latest
     container:
       image: ${{ vars.CI_IMAGE }}
     steps:
-      - uses: actions/checkout@v3
+      - name: Checkout sources
+        uses: actions/checkout@v4
 
-      # uncomment if needed
-      # - name: fmt
-      #   run: |
-      #     cargo fmt --all --check
+      - name: Install Rust nightly toolchain
+        uses: actions-rs/toolchain@v1
+        with:
+          profile: minimal
+          toolchain: stable
+          override: true
+          components: rustfmt
```

Comment:
```suggestion
```
This part is not needed since we run the action in [the `ci-unified`docker image](https://github.com/paritytech/scripts/tree/master/dockerfiles/ci-unified). Also the `actions-rs` org is archived and the actions are not maintained anymore.